appdirectory/register/templates/register/register.html

```diff
@@ -0,0 +1,4 @@
+{% extends "cart/base.html" %}
+{% block content %}
+    <div class=""
```

Comment:
<div class=""></div>?
Do you need any special template logic to load the form template you're passing in?